src/main/java/org/broadinstitute/hellbender/tools/exome/gcbias/CorrectGCBias.java

```diff
@@ -0,0 +1,85 @@
+package org.broadinstitute.hellbender.tools.exome.gcbias;
+
+import org.broadinstitute.hellbender.cmdline.*;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.exome.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ */
+@CommandLineProgramProperties(
+        oneLineSummary = "Correct for per-sample GC bias effects.",
+        summary = "Correct for per-sample GC bias effects.",
```

Comment:
Is the output (i.e. asjusted counts) float or int ?

Also, can you put some of this in the summary field?  I would like it to show up in the ``--help``